examples/social-media/common/schemas.yaml

```diff
@@ -0,0 +1,38 @@
+commentLikedPayload: 
+  type: object
+  title: commentLikedPayload
+  additionalProperties: false
+  properties:
+    commentId: 
+      allOf: 
+        - $ref: '#commentId'
+        - description: Id of the comment that was liked
+likeCommentPayload: 
+  type: object
+  title: likeCommentPayload
+  additionalProperties: false
+  properties:
+    commentId: 
+      allOf: 
+        - $ref: '#/commentId'
+        - description: Id of the comment that should be liked
+commentChangedPayload: 
+  type: object
+  title: commentChangedPayload
+  additionalProperties: false
+  properties:
+    commentId: 
+      allOf: 
+        - $ref: '#/commentId'
+        - description: Id of the comment that was changed, such as when someone liked it.
+updateCommentLikesPayload:
+  type: object
+  title: updateCommentLikesPayload
+  additionalProperties: false
+  properties:
+    commentId: 
```

Comment:
I think a `likeCount` should be included as well.

Reply:
There are definitely multiple things that could be added to make the system more believable. However, we gotta stop somewhere, and I just decided to take a minimal approach.

I guess my question is: would this minimal payload not show what is needed? 🤔

Reply:
It requires a separate call to update the count, and there are race conditions. I don't feel strongly as it's just an example.

Reply:
ideal example would be actually something that we can use in code generators to generate real project

Reply:
@derberg so you also think we should add more details to the messages?

Reply:
hahaha, when I clicked on `Add single comment` I had a quick thought in my head "this one can be confusing" 😄 

basically, when I generate code with these examples, in frontend app when I add logic to react on event about comments count, I will not have access to actually the count information. Thus from a code generation perspective, this is not a full example.

Does it make sense?

Reply:
Hmm, that is a good point...

Reply:
Alright, I have added a `likedBy` property to the `likeCommentPayload`, and `likeCount` to the two schemas `commentChangedPayload`, `updateCommentLikesPayload`.

Not sure if `likedBy` makes sense 🤔 Maybe better to be left out. What do you think?